### ISSUE TITLE
Output of vroom::vroom does not display properly in Quarto HTML

### ISSUE THREAD

**Issue by allefeld** [tidyverse/vroom#537]
This is what it looks like in an R session in a terminal:

![image](https://github.com/tidyverse/vroom/assets/1738353/f113e650-a5f9-4b30-b319-82a3f2dd506b)

This is what it looks like in a Quarto document rendered to HTML:

![image](https://github.com/tidyverse/vroom/assets/1738353/aa48be6a-8250-4441-b107-684af8d7400e)

I obviously don't expect the terminal colors to be reproduced. But beyond that, output seems to be out of order, line breaks are not respected etc.

In case this is a bug in Quarto and not in vroom, I'm tagging @cscheid.

**Comment by cscheid** [comment]
@allefeld Can you provide a minimal example I can run myself?

**Comment by allefeld** [comment]
Here it is:

File `test.qmd`:

````markdown
```{r}
df <- vroom::vroom("test.csv")
```
````

File `test.csv` (intentionally broken to trigger messages from `vroom`:

````markdown
,Column 2,12
1,2,
,3,
````

Screenshot from rendered `test.html`:

![image](https://github.com/tidyverse/vroom/assets/1738353/9d7f9e7d-144b-4651-a600-49961911b49b)

Screenshot from same code run in a terminal:

![image](https://github.com/tidyverse/vroom/assets/1738353/c282c838-c60b-4503-819d-31d8382d85c9)

Cell output HTML code:

````html
<div class="cell-output cell-output-stderr">
<pre><code>New names:
Rows: 2 Columns: 3
── Column specification
──────────────────────────────────────────────────────── Delimiter: &quot;,&quot; dbl
(2): ...1, Column 2 lgl (1): 12
ℹ Use `spec()` to retrieve the full column specification for this data. ℹ
Specify the column types or set `show_col_types = FALSE` to quiet this message.
• `` -&gt; `...1`</code></pre>
</div>
````

**Comment by allefeld** [comment]
Agreed, seems like vroom creates its output in a strange way. Control characters to move the cursor?
